pkg/apiserver/filters/audit.go

```diff
@@ -125,6 +124,18 @@ func WithAudit(handler http.Handler, attributeGetter RequestAttributeGetter, out
 	})
 }
 
+func auditStringSlice(inList []string) string {
+	if len(inList) == 0 {
+		return ""
+	}
+
+	quotedElements := make([]string, len(inList))
+	for i, in := range inList {
+		quotedElements[i] = fmt.Sprintf("%q", in)
+	}
+	return strings.Join(quotedElements, ", ")
```

Comment:
> "\"" + strings.Join(inList, "\", \"") + "\"" ;-)

That's not obvious to me.  Each individual string has quotes around it already, so in the log you get `"\"group1\", \"group2\""`.  You want `"\"\"group1\"\", \"\"group2\"\""`?


For instance, `user="system:apiserver" groups="\"system:masters\""`